test/test.js

```diff
@@ -70,6 +70,60 @@ module.exports['html/first.html'] = '<p>\\n  \\'first\\'\\n</p>';
 module.exports['html/second.html'] = '<p>\\n  "second"\\n</p>';\n`) throw Error()
 
 /**
+ * With path prefix
+ */
+
+stream = plugin({concat: 'view.js', prefix: 'templates'})
+
+for (const file of files()) {
+  stream._transform(file, null, onerror)
+}
+
+stream._flush(onerror)
+
+buffer = stream._readableState.buffer
+
+if (buffer.length !== 1) throw Error()
+
+result = buffer[0]
+
+if (!result) throw Error()
+
+if (result.relative !== 'view.js') throw Error()
+
+if (result.contents.toString() !== `'use strict';
+module.exports = Object.create(null);
+module.exports['templates/html/first.html'] = '<p>\\n  \\'first\\'\\n</p>';
+module.exports['templates/html/second.html'] = '<p>\\n  "second"\\n</p>';\n`) throw Error()
+
+/**
+ * With global
+ */
+
+stream = plugin({concat: 'view.js', global: 'window.templates'})
+
+for (const file of files()) {
+  stream._transform(file, null, onerror)
+}
+
+stream._flush(onerror)
+
+buffer = stream._readableState.buffer
+
+if (buffer.length !== 1) throw Error()
+
+result = buffer[0]
+
+if (!result) throw Error()
+
+if (result.relative !== 'view.js') throw Error()
+
+if (result.contents.toString() !== `'use strict';
+window.templates = Object.create(null);
+window.templates['html/first.html'] = '<p>\\n  \\'first\\'\\n</p>';
+window.templates['html/second.html'] = '<p>\\n  "second"\\n</p>';\n`) throw Error()
```

Comment:
The difference here is the `window.templates` variable assignment, rather than `module.exports`